bot/handlers/notify_groups/invitations.py

```diff
@@ -241,11 +243,18 @@ def reply_to_invite(update, context):
     """
     update = update.callback_query
     update.answer()
-    reply, notify_group_id = update.data.split("_")
+    reply, notify_group_id, invitation_id = update.data.split("_")
     curr_user_id = update.from_user.id
     chat_id = update.message.chat.id
     curr_username = (f"@{update.from_user.username}"
                      if update.from_user.username else None)
+    notify_group_invitation = context.chat_data[invitation_id]
+    
+    # Let the user know we are processing their request.
+    notify_group_invitation["inv_msg"].edit_text(
```

Comment:
Not a perfect solution, not very pretty, ideally the other users shouldn't be able to see that it's processing for one user. Create an issue regarding better processing mechanics when user tries to accept/decline and ensure that doesn't intrude on other people's command experience